app/src/main/java/org/mifos/mobile/ui/activities/RegistrationActivity.kt

```diff
@@ -13,4 +15,16 @@ class RegistrationActivity : BaseActivity() {
         setContentView(R.layout.activity_registration)
         replaceFragment(RegistrationFragment.newInstance(), false, R.id.container)
     }
+
+    override fun onBackPressed() {
+        MaterialDialog.Builder().init(this)
```

Comment:
can we call the override method too. `super.onBackPresses`

Reply:
@miPlodder i have called it on line 24. Calling it alongwith dialog would just backpress so i kept it in positive button.

Reply:
okay, i missed it. Good job